src/plugins/cal-events/cal-events.js

```diff
@@ -80,22 +80,36 @@ var componentName = "wb-calevt",
 	},
 
 	processEvents = function( $elm ) {
-		var settings = $.extend( {}, window[ componentName ], $elm.data( dataAttr ) ),
-			year, month, events, minDate, containerId, $container;
+		var settings = $.extend( {}, window[ componentName ], $elm.data( dataAttr ) ), // Ability to extend the calendar
+			year, month, events, minDate, maxDate, containerId, $container,
+			currDate = new Date(),
+			currYear = currDate.getFullYear(),
+			currMonth = currDate.getMonth();
 
 		events = getEvents( $elm );
 		containerId = $elm.data( "calevtSrc" );
 		$container = $( "#" + containerId ).addClass( componentName + "-cal" );
 
 		minDate = events.minDate;
-		year = settings.year || minDate.getFullYear();
-		month = settings.month || minDate.getMonth();
+		maxDate = events.maxDate;
+
+		if ( minDate.getTime() > currDate.getTime() ) {
```

Comment:
The fonction getEvent returns the object {minDate: Thu Mar 23 2017 00:00:00 GMT-0400 (EDT), maxDate: Wed Mar 11 2020 00:00:00 GMT-0400 (EDT), ...} when the multi-Day event has been set to March 22nd, 2017 to April 26th, 2017. And we want minDate to be the date of today. This is the problem. Does getEvents work correctly?